packs/feats/fresh-ingredients.json

```
@@ -31,7 +31,7 @@
         "rules": [
             {
                 "key": "FlatModifier",
-                "label": "Spend a batch of herbs in wilderness",
+                "label": "PF2E.SpecificRule.Herbalist.FreshIngredientsWilderness.FlatModifierLabel",
```

```suggestion
                "label": "PF2E.SkillVariant.Wilderness",
```
I'll add this localization in another update